INSTALL.md

```diff
@@ -146,7 +146,7 @@ webhooks:
         name: autocert
         namespace: step
         path: "/mutate"
-      caBundle: $(cat $(step path)/certs/root_ca.crt | base64)
+      caBundle: $(cat $(step path)/certs/root_ca.crt | base64 -w 0)
```

Comment:
Can you remove the `-w 0` from here, as it does not work on a macOS, and it really doesn't make any difference here. The one in the `init/autocert.sh` is ok as it runs in a linux container.

Reply:
The reason I added it there is because my Linux distro, Ubuntu 18.04, by default wraps lines. Options: 
* Revert
* Add the removal of \n, \r here
* Add code that detects OS (like here: https://stackoverflow.com/questions/3466166/how-to-check-if-running-in-cygwin-mac-or-linux)

Reply:
@Kristof-Mattei I've tested with an ubuntu, and now I see your issue, you should use one of these solutions:
* `cat $(step path)/certs/root_ca.crt | base64 |  tr -d '\n'`
* `cat $(step path)/certs/root_ca.crt | step base64`

I leave it to you to pick one.